python/factorialIterativo.py

```diff
@@ -0,0 +1,19 @@
+
+"""
+ Program performs the factorial of n, in iterative form. (n!)
+"""
+
+def factorial(n):
+    result = 1
+ 
+    for i in range(n):
+        result *= (i + 1)
+ 
+    return result
+  
+
+
+value = input("Enter with value: ")
```

Comment:
Handle negative input by printing that factorial of negative number is not defined.